src/update_engine/omaha_hash_calculator.h

```diff
@@ -10,7 +10,8 @@
 #include <vector>
 
 #include <glog/logging.h>
-#include <openssl/sha.h>
+#include <openssl/evp.h>
```

Comment:
Is this include file here necessary? I don't see anything from OpenSSL in this header.